### DISCUSSION TITLE
Sunset subject:organizationalUnitName

### DISCUSSION THREAD

**Discussion by VenkatKandhari1351** [pkic/community#24] [Identity]
Hi All:
A quick question

If a Certificate Signing Request (CSR) contains Organization Unit Name then would the Certificate Authorities listed below would
Reject the certificate signing request

OR

Ignore the Organization Unit in the CSR and still the certificate?

**Certificate Authorities**
Amazon, Buypass, Certum (Asseco), Chunghwa Telecom, D-TRUST, DigiCert, Disig, eMudhra, Entrust, Firmaprofesional, GDCA, GlobalSign, GoDaddy, HARICA, iTrusChina, Izenpe, JPRS, Kamu SM, Let’s Encrypt / ISRG, OISTE, Sectigo, Telia Company, TrustCor, SecureTrust, Visa

Regards,
Venkat

**Comment by vanbroup** [comment]
Entrust will ignore OU and continue to issue the certificate.

**Reply by sandorszoke** [reply]
Microsec will ignore the OU, but the Subscriber shall accept the modified data before the issuance

**Comment by barrini** [comment]
https://sectigo.com/resource-library/oufield

**Reply by VenkatKandhari1351** [reply]
Barrini: Does it mean that from Jul 1st 2022 or Sep 1st 2022, Sectigo would ignore a OU field in Certificate Signing Request (CSR) and still would sign the certificate? i.e. if CSR contains OU field, Sectigo won't reject CSR but will go ahead and sign it

**Reply by VenkatKandhari1351** [reply]
Hi All:
Can we have behavior from other CA providers please?


Regards,
Venkat

**Comment by YairEisenstein** [comment]
Comsign will ignore the OU and continue to issue the certificate

**Comment by barrini** [comment]
Hi,

The CABF set the date to September 1st but in Sectigo we´re doing it since
July 1st and yes, we´ll ignore the OU.

Regards

**Comment by ja-saez** [comment]
Izenpe ignores the OU and continue to issue the certificate since last October

**Comment by Pabloanf** [comment]
ANF AC will ignore the OU and continue to issue the certificate.

**Reply by VenkatKandhari1351** [reply]
Thank you all. Appreciate all the responses.
This is a great info but is not complete yet. 

This info would be useful for many technological solutions.

Please respond with your respective CA behavior

Will your CA ignore the OU field and issue the certificate or Reject it

**Reply by chemalogo** [reply]
Firmaprofesional ignores the OU and continue to issue the certificate.
